qcodes/tests/dataset/test_data_set_cache.py

```diff
@@ -331,6 +333,150 @@ def test_cache_complex_array_param_in_1d(experiment, DAC, channel_array_instrume
                                         dataset.cache.data())
 
 
+@pytest.mark.parametrize("bg_writing", [True, False])
+@pytest.mark.parametrize("cache_too_small", [True, False])
```

Comment:
it seems that we need a parameterization here for an interrupted dataset. i think expectations are the same